lib/v2/vcb-s/index.js

```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootUrl = 'https://vcb-s.com';
+const postsAPIUrl = `${rootUrl}/wp-json/wp/v2/posts`;
+
+module.exports = async (ctx) => {
+    const limit = ctx.params.limit ?? 7;
+    const url = `${postsAPIUrl}?per_page=${limit}&_embed`;
+
+    const response = await got.get(url);
+    if (typeof response.data === 'string') {
+        response.data = JSON.parse(response.body.trim());
+    }
+    const data = response.data;
+
+    const items = data.map((item) => {
+        let description = item.content.rendered
+            .replace(/<div.+?abh_box[\w\W]+/g, '')
+            .replace(/<p.+?medie-info-switch[\w\W]+/g, '')
+            .replace(/<div.+?dw-box-download.+?>([\w\W]+?)<\/div>/g, '<pre>$1</pre>');
```

Comment:
Use `.` instead of `[\w\W]`
https://rules.sonarsource.com/javascript/RSPEC-6353

Reply:
use `.`  not match my need, content has some line break char

Reply:
Has fix, i miss the `s` flag